pipeline/reduce.py

```diff
@@ -18,7 +18,7 @@
 
 
 @task
-def save_query(region, part_type):
+def save_query(segment):
```

Comment:
Maybe a better name?  Something that's connected to the query itself like "revenue_by_supplier" or something?

Reply:
I went with ``unshipped_orders_by_revenue`` -- further suggestions are welcome